### DISCUSSION TITLE
Get Ory's main docker images listed as trusted content

### DISCUSSION THREAD

**Discussion by alantbarlow** [ory/.github#12] [Ideas]
Ory Hydra, Kratos, Keto, and Oathkeeper should all be listed on docker as trusted content. Ideally they get listed alongside Traefik and Caddy as [official images](https://docs.docker.com/docker-hub/official_images/), but getting them listed as [verified images](https://www.docker.com/partners/programs/) could also work.

I believe doing so would help boost the popularity of the images as it would help them become more visible. It would also help the users know they are using the official images when they are setting up their authentication servers. This will also provide the benefit of the images not counting towards the user's pull limit.

**Comment by vinckr** [comment]
Hello Alan, 
that would be neat, thanks for pointing it out! 
I will see to get something going.

official images 
- add PR for dockerfile here https://github.com/docker-library/official-images
- add docs here https://github.com/docker-library/docs

verified 
- check if Ory is in verified publisher program
